OmniStudio-Components/main/default/omniScripts/EDRD_CaseNewRenewForm_English_11.os-meta.xml

```diff
@@ -2148,7 +2148,7 @@
   }, {
     &quot;value&quot; : false,
     &quot;type&quot; : &quot;Requirement&quot;,
-    &quot;text&quot; : &quot;The selected patient does not exist in the case management system. Please contact PHSA PSP to set up the patient account (psp@phsa.ca).&quot;,
+    &quot;text&quot; : &quot;[5:28 PM] Gupta, Komal D. Phn Not Found&quot;,
```

Comment:
@deepakmulamalla @Vinayak-accenture Could we get this part updated, looks like a copy and paste error.  Thanks!